cves/CVE-2013-0863.yml

```diff
@@ -355,11 +365,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
-nickname_instructions: |
+CWE: 119
+CWE_note: Improper Restriction of Operations within the Bounds of a memory buffer.
```

Comment:
I'd explain why you chose this in the note, rather than just pasting the title as I don't think that was the intent.